src/test/java/com/epam/test_generator/dto/ProjectTransformerTest.java

```diff
@@ -0,0 +1,151 @@
+        Project expectedProject = new Project();
+        expectedProject.setName(NAME);
+        expectedProject.setDescription(DESCRIPTION);
+        expectedProject.setJiraKey(JIRA_KEY);
+
+        ProjectCreateDTO projectCreateDTO = new ProjectCreateDTO();
+        projectCreateDTO.setName(NAME);
+        projectCreateDTO.setDescription(DESCRIPTION);
+        projectCreateDTO.setJiraKey(JIRA_KEY);
+
+        Project resultProject = projectTransformer.fromDto(projectCreateDTO);
+        Assert.assertEquals(expectedProject, resultProject);
+    }
+
+    @Test
+    public void toDTO_Project_Success() {
+        Project project = new Project();
+        project.setId(ID);
+        project.setName(NAME);
+        project.setDescription(DESCRIPTION);
+        project.setJiraKey(JIRA_KEY);
+        project.setActive(true);
+        project.setUsers(Collections.emptySet());
+
+        ProjectDTO expectedProjectDTO = new ProjectDTO();
+        expectedProjectDTO.setId(ID);
+        expectedProjectDTO.setName(NAME);
+        expectedProjectDTO.setDescription(DESCRIPTION);
+        expectedProjectDTO.setJiraKey(JIRA_KEY);
+        expectedProjectDTO.setActive(true);
+        expectedProjectDTO.setUsers(Collections.emptySet());
+
+        ProjectDTO resultProjectDTO = projectTransformer.toDto(project);
+        Assert.assertEquals(expectedProjectDTO, resultProjectDTO);
+    }
+
+    @Test
+    public void toFullDTO_Project_Success() {
+        when(suitTransformer.toDto(any(Suit.class))).thenReturn(suitDTO);
+        Project project = new Project();
+        project.setId(ID);
+        project.setName(NAME);
+        project.setDescription(DESCRIPTION);
+        project.setJiraKey(JIRA_KEY);
+        project.setActive(true);
+        project.setUsers(Collections.emptySet());
+        project.setSuits(Collections.singletonList(suit));
+
+        ProjectFullDTO expectedProjectDTO = new ProjectFullDTO();
+        expectedProjectDTO.setId(ID);
+        expectedProjectDTO.setName(NAME);
+        expectedProjectDTO.setDescription(DESCRIPTION);
+        expectedProjectDTO.setJiraKey(JIRA_KEY);
+        expectedProjectDTO.setActive(true);
+        expectedProjectDTO.setUsers(Collections.emptySet());
+        expectedProjectDTO.setSuits(Collections.singletonList(suitDTO));
+
+        ProjectFullDTO resultProjectDTO = projectTransformer.toFullDto(project);
+        Assert.assertEquals(expectedProjectDTO, resultProjectDTO);
+    }
+
+    @Test
+    public void updateFromDto_ProjectUpdateDTO_Project_Success() {
+        when(suitTransformer.fromDto(any(SuitDTO.class))).thenReturn(suit);
+
+        Project expectedProject = new Project();
+        expectedProject.setName(NAME);
+        expectedProject.setDescription(DESCRIPTION);
+        expectedProject.setJiraKey(JIRA_KEY);
+        expectedProject.setActive(true);
+        expectedProject.setUsers(Collections.emptySet());
+        expectedProject.setSuits(Collections.singletonList(suit));
```

Comment:
You have similar declaration of a project more then 2 times. It looks like copy/paste.
May be you should move duplication to the setUp() method?